WTWSMS/decisions/conversion_decisions.txt

```diff
@@ -125,7 +125,6 @@ decisions = {
 					NOT = { culture = ROOT }
 				}
 			}
-			holy_order = no
```

Comment:
It doesn't because converting a province in Mongolia to France is FAR more unrealistic than a Frenchman not becoming Mongolian after spending his entire life in Mongolia. Culture does not effect his religion.